qcengine/programs/tests/test_standard_suite_hf.py

```diff
@@ -37,6 +37,12 @@ def nh2():
     [
         pytest.param("cfour", "aug-pvdz", {"scf_conv": 12}, marks=testing.using_cfour),
         pytest.param("cfour", "aug-pvdz", {}, marks=testing.using_cfour),
+        pytest.param(
+            "entos",
+            "aug-cc-pVDZ",
+            {"coulomb_method": "direct_4idx", "exchange_method": "direct_4idx"},
+            marks=testing.using_entos
+        ),
```

Comment:
Awesome; didn't know entos actually supported "normal" integrals.

Reply:
Yup! It is a little slow. I did try to use a giant density fitting basis, but `aug-cc-pVTZ-JKFIT` didn't cut it and there is no `QZ` version.